docker/entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Start the REST API with 4 workers
+echo "Starting REST mode..."
+gunicorn "keep.api.api:get_app(mode='rest')" \
+  --bind 0.0.0.0:8080 \
+  --workers 4 \
+  -k uvicorn.workers.UvicornWorker \
+  -c /venv/lib/python3.11/site-packages/keep/api/config.py &
+
+# Check if REDIS is true, and start the worker mode if it is
+# I'm using random port 1234 because it doesn't matter
+if [ "$REDIS" = "true" ]; then
+  echo "Starting worker mode..."
```

Comment:
Dunno why this change but small comment: arq is a cli tool and can be executed independently. Guincorn is redundant in this case ❤️

Reply:
Yea I know I didn't to keep it a minimal change.

The full context is wide by this specific change is because if there are lot of alerts in the queue, the HTTP requests are throttled and not served, which I wanted to change by separating between workers and REST processes.